roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -21,7 +21,7 @@
 
 - name: Stop if unknown OS
   assert:
-    that: ansible_os_family in ['RedHat', 'CentOS', 'Fedora', 'Ubuntu', 'Debian', 'CoreOS', 'Container Linux by CoreOS', 'openSUSE Leap', 'openSUSE Tumbleweed', 'ClearLinux']
+    that: ansible_distribution in ['RedHat', 'CentOS', 'Fedora', 'Ubuntu', 'Debian', 'CoreOS', 'Container Linux by CoreOS', 'openSUSE Leap', 'openSUSE Tumbleweed', 'ClearLinux']
```

Comment:
@fdegir thanks a lot for fixing my mistake. might i ask for a small change. either we change `ClearLinux` to `Clear Linux OS` or add another line with `that: ansible_os_family in ["ClearLinux"]

Reply:
done. Thanks for the review.